proof/access-control/ARM/ArchInterrupt_AC.thy

```diff
@@ -75,7 +74,7 @@ global_interpretation Interrupt_AC_1?: Interrupt_AC_1 "arch_authorised_irq_ctl_i
 proof goal_cases
   interpret Arch .
   case 1 show ?case
-    by (unfold_locales; (fact Interrupt_AC_assms)?)
+    by (unfold_locales; fact Interrupt_AC_assms)
```

Comment:
you can keep this change, but should know the reason for `(fact Interrupt_AC_assms)?`. The reason that's useful is while developing, when this instantiation errors, you can see which fact you're missing from your assumption; with the non-`?` one, you get "proof command failed".

Reply:
This is also how I use them, can re-add if we want this artifact to exist outside of development